enatega-multivendor-admin/.env

```
@@ -1,3 +1,2 @@
-NEXT_PUBLIC_SERVER_URL="https://aws-server.enatega.com/"
-NEXT_PUBLIC_WS_SERVER_URL="wss://aws-server.enatega.com/"
-
+NEXT_PUBLIC_SERVER_URL="http://localhost:8001/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)